.travis.yml

```diff
@@ -28,6 +28,12 @@ language: python
 python:
   - "3.4"
 
+addons:
+  postgresql: "9.4"
+
+env:
+  - SQLALCHEMY_DATABASE_URI=postgres://postgres@localhost:5432/claims
+
```

Comment:
If we change the database name, let's use the same name in `docker-compose.yml` too?  

BTW `claimstore` may be more appropriate DB name, with `claims` being one of its tables.